Min-Max_Guide.md

```diff
@@ -276,28 +276,31 @@ An alternate strategy for Spring 1 would be to ignore all of the NPCs and focus
 Another alternative is to exploit RNG to farm clay, but this is illegal in our ruleset.
 
 - Take the Parsnip Seeds.
-- Immediately chop down 9 trees until foraging level 1 (8 1/3 trees needed). We want to get level 1 before we leave to go foraging so that we have a chance to get silver-quality Spring Onions. (Focus on **only** chopping down trees, and begin in the direction of your path to the forest so that some of the weeds you clear are helpful for that.)
-  - Only chop down [Maple Trees](https://stardewvalleywiki.com/Maple_Tree) and [Pine Trees](https://stardewvalleywiki.com/Pine_Tree), not [Oak Trees](https://stardewvalleywiki.com/Oak_Tree). (We won't get any [Acorns](https://stardewvalleywiki.com/Acorn) for felling an Oak Tree until Spring 2 and acorns are precious.)
+- Set your UI Scale and Zoom Level to 75%.
+- Immediately chop down nine trees until foraging level 1 (8 1/3 trees needed). We want to get level 1 before we leave to go foraging so that we have a chance to get silver-quality Spring Onions. (Focus on **only** chopping down trees, and begin in the direction of your path to the forest so that some of the weeds you clear are helpful for that.)
+  - Only chop down [Maple Trees](https://stardewvalleywiki.com/Maple_Tree) and [Pine Trees](https://stardewvalleywiki.com/Pine_Tree), not the egg-shaped [Oak Trees](https://stardewvalleywiki.com/Oak_Tree). (We won't get any [Acorns](https://stardewvalleywiki.com/Acorn) for felling an Oak Tree until Spring 2, and acorns are precious.)
   - For the same reason, do not axe/hoe any seeds in the ground until Spring 2, or they will be destroyed.
-  - Do not chop the stumps; only chop a tree until it falls, then move on to the next one. (It yields more wood and more foraging XP to do it this way.)
-  - As soon as a tree starts to fall, walk towards where the wood will spawn and immediately pause the game. This prevents in-game time from passing, but the tree falling animation will still complete. Additionally, all of the wood will automatically be collected by the player while the game is still paused. (You should use this strategy whenever you are chopping trees and not bothering to kill the stumps.)
+  - Do not chop the stumps; only chop a tree until it falls, limiting it to 10 strikes, then move on to the next one. (It yields more wood and more foraging XP to do it this way.) If you lose count, check your skills for foraging level 1 in the menu.
+  - As soon as a tree starts to fall, walk towards where the wood will spawn and immediately pause the game. This approach prevents in-game time from passing, but the tree-falling animation will still be complete. Additionally, the player will automatically collect all of the wood while the game is still paused. (You should use this strategy when chopping trees and not bothering to kill the stumps.)
```

Comment:
why are we duplicating information about not killing the stumps

Reply:
I was making these changes in the browser during travel as my machine didn't have git set up. I'll address it.